Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue2653.cs

```diff
@@ -73,17 +87,45 @@ protected override void Init()
 						layout.LowerChild(bv);
 					}),  HeightRequest = 45},
 					layout,
-					new Button(){ Text = TestForButtonClicked, Command = new Command(() =>
+					new Button()
 					{
-						if(!layout.Children.Contains(bv))
+						Text = TestForButtonClicked, Command = new Command(() =>
 						{
-							labelInstructions.Text = Success;
-						}
-						else
+							if(bv == null)
+							{
+								labelInstructions.Text = String.Empty;
+							}
+							else if(!layout.Children.Contains(bv))
+							{
+								labelInstructions.Text = Success;
+							}
+							else
+							{
+								labelInstructions.Text = BoxViewIsOverlappingButton;
+							}
+						}),
+						HeightRequest = 45
+					},
+					new Button()
+					{
+						Text = ClickShouldAddText, Command = new Command(() =>
 						{
-							labelInstructions.Text = BoxViewIsOverlappingButton;
-						}
-					}),  HeightRequest = 45}
+							layout.Children.Insert(0, new Label());
+							layout.Children.Add(new Grid()
+							{
+								Children =
+								{
+									new Label()
+									{
+										Margin = 10,
+										Text = "If you can't see me test has failed",
```

Comment:
If I can't see this when running the test, how will I know that not seeing it means the test has failed?

Reply:
I mainly added that just as default text and didn't account for manual runners. I'll add some additional text to the button about expectation